plugins/woocommerce/includes/class-wc-mobile-messaging-handler.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * Prepares formatted mobile deep link navigation link for order mails
+ *
+ * @package WooCommerce\Emails
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Mobile_Messaging_Handler
+ */
+class WC_Mobile_Messaging_Handler {
```

Comment:
Ah, I wasn't aware, or at least had forgotten about that guidance. I guess it makes sense for these functions since they're encapsulated as a class. 👍